roles/pulp3/README.md

```diff
@@ -35,6 +35,8 @@ Role Variables:
   the systemd module's "state" directive. Defaults to "started".
 * `pulp_wsgi_enabled` This variable can be configured with any of the states allowed
   by the systemd module's "enabled" directive. Defaults to "true."
+* `pulp_port` Set the port the API server is served from. Defaults to '8000'.
+* `pulp_host` Set the host the API server is served from. Defaults to 'localhost'.
```

Comment:
Should 8000 and localhost be in backticks for better formating?

Reply:
For now, I just followed the README standard of using quotes. I agree that is a good idea.